pyk/src/pyk/kcfg.py

```diff
@@ -322,7 +366,7 @@ def _short_label(label):
         for node in self.nodes:
             nodeAttrs = self.node_attrs(node.id)
             classAttrs = ' '.join(nodeAttrs)
-            label = shorten_hashes(node.id) + (classAttrs and ' ' + classAttrs)
+            label = self.short_id(node) + (classAttrs and ' ' + classAttrs)
```

Comment:
The label should be based on `node_short_info` if we want to display all aliases.